tests/viscoelastic_plastic_numerical_elastic_time_step_yield/statistics

```diff
@@ -15,5 +15,5 @@
 # 15: Schur complement iterations in Stokes preconditioner
 # 16: RMS velocity (m/year)
 # 17: Max. velocity (m/year)
-0 0.000000000000e+00 0.000000000000e+00 400 3803 441 5043 1 0  0  0  0 26 28 28 2.57523019e-02 4.43529491e-02 
-1 1.000000000000e+03 1.000000000000e+03 400 3803 441 5043 1 0 11 11 12 14 16 16 2.57523019e-02 4.43549287e-02 
+0 0.000000000000e+00 0.000000000000e+00 400 3803 441 5043 1 0  0  0  0         26 28 28 2.57523019e-02 4.43529491e-02 
+1 1.000000000000e+03 1.000000000000e+03 400 3803 441 5043 1 0 11 11 12 4294967295  0  0 2.57523019e-02 4.43529491e-02 
```

Comment:
unrelated, but do we really output -1 here for the number of Stokes iterations?

Reply:
I think that is a bug with a specific deal.II version (9.0) if there are no Stokes iterations. We then output `invalid_unsigned_int`. I saw this a while ago, but never got around to fixing it. I think it is fixed for deal.II 9.1.